src/seedu/addressbook/Main.java

```diff
@@ -125,5 +129,12 @@ private StorageFile initializeStorage(String[] launchArgs) throws InvalidStorage
         return isStorageFileSpecifiedByUser ? new StorageFile(launchArgs[0]) : new StorageFile();
     }
 
+    private void checkStorageFileExists() throws FileNotFoundException {
+        if (!DEFAULT_STORAGE_FILE.exists()) {
+            throw new FileNotFoundException("File not found");
+        }
```

Comment:
Throwing an exception from this method seems like an overkill. A `boolean isStorageFilePresent()` is more suitable?